sksurv/datasets/base.py

```diff
@@ -474,9 +475,83 @@ def load_bmt():
     .. [1] https://doi.org/10.1038/sj.bmt.1705727
            Scrucca, L., Santucci, A. & Aversa, F.:
            "Competing risk analysis using R: an easy guide for clinicians. Bone Marrow Transplant 40, 381–387 (2007)"
+
     .. [2] https://luca-scr.github.io/R/bmt.csv
     """
     full_path = _get_data_path("bmt.arff")
     data = loadarff(full_path)
     data["ftime"] = data["ftime"].astype(int)
     return get_x_y(data, attr_labels=["status", "ftime"], competing_risks=True)
+
+
+def load_cgvhd():
+    r"""Load and return data from multicentre randomized clinical trial
+    initiated for patients with a myeloid malignancy who were to
+    undergo an allogeneic bone marrow transplant.
+
+    The dataset has 100 samples and 4 features:::
+
+        1. dx: Diagnosis: AML=acute myeloid leukaemia, CML=chronic myeloid leukaemia, MDS=myelodysplastic syndrome
+        2. tx: Randomized treatment: BM=cell harvested from the bone marrow, PB=cell harvested from the peripheral blood
+        3. extent: Extent of disease: L=limited, E=extensive
+        5. age: Age (years)
+
+    The rest of the columns correspond to outcome variables:::
+
+        6.  survtime: Time from date of transplant to death or last follow-up (Years)
+        7.  reltime: Time from date of transplant to relapse or last follow-up (Years)
+        8.  agvhtime: Time from date of transplant to acute GVHD or last follow-up (Years)
+        9.  cgvhtime: Time from date of transplant to chronic GVHD or last follow-up (Years)
+        10. stat Status: 1=Dead, 0=Alive
+        11. rcens Relapse: 1=Yes, 0=No
+        4.  agvhdgd: Grade of acute GVHD
+        12. agvh: Acute GVHD: 1=Yes, 0=No
+        13. cgvh: Chronic GVHD: 1=Yes, 0=No
```

Comment:
A [table](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#tables) might more suitable to describe the dataset. Headers could be "Index", "Name", "Description", "Encoding".